README.md

```diff
@@ -60,48 +60,26 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Portofilo] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+> This is my portofilo for mobile version.
 
-**[your_project__name]** is a...
+**[Portofilo]** is a project that shows how the website is shown on mobile.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
 > Describe the tech stack and include only the relevant sections that apply to your project.
```

Comment:
Please remove this line because it's only an instruction of what you can put here